antarest/study/storage/variantstudy/model/command/create_cluster.py

```diff
@@ -53,10 +53,17 @@ class CreateCluster(ICommand):
 
     @field_validator("cluster_name", mode="before")
     def validate_cluster_name(cls, val: str) -> str:
-        valid_name = transform_name_to_id(val, lower=False)
-        if valid_name != val:
+        to_return = transform_name_to_id(val)
+        if not to_return:
             raise ValueError("Cluster name must only contains [a-zA-Z0-9],&,-,_,(,) characters")
-        return val
+        return to_return
+
+    @field_validator("parameters", mode="before")
+    def lower_cluster_group_and_names(cls, params: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
```

Comment:
This dict makes it really hard to maintain. We should use a pydantic model instead to make expectations explicit.